app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :posts
+  has_many :likes
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
+  def update_comments_counter
+    update(comments_counter: comments.count)
+  end
+
```

Comment:
- [ ]  The methods used here are not correct, you should use proper methods names as we are doing methods for posts.  Check in the requirements : 
![image](https://github.com/ikoote1/EarnChat/assets/43877689/d08d1edb-d393-4b69-b530-b6b8c4754fff)